Source/doomstat.h

```diff
@@ -79,6 +79,8 @@ extern int demo_version;           // killough 7/19/98: Version of demo
 
 #define demo_compatibility (demo_version < 200) /* killough 11/98: macroized */
 
+#define mbf21 (demo_version == 221)
```

Comment:
I'm not asking for a full review, but what do you think of this macro?

Also, how to organize the work on mbf21, one branch or a dozen PRs?

Reply:
I am alright with this macro. I think dsda uses the same, right? This will make it easier to port code over.

I'd prefer the work on MBF21 to happen on a separate branch. We don't have to merge all in one go, though, but logical units. This preparation branch, for example, would be one such unit. However, I will not merge it yet, because I'd first like to push a maintenance release next week.

Reply:
So, am I continuing to work on this branch or are we creating a new one? I need `doomstat.h` changes from this branch.

Reply:
> So, am I continuing to work on this branch or are we creating a new one? I need `doomstat.h` changes from this branch.

Why should we create a new branch?

Reply:
> Why should we create a new branch?

> I'd prefer the work on MBF21 to happen on a separate branch.

Sorry, I guess I don't understand.

Reply:
Well, but this PR is its own separate branch already. 😉

Reply:
My point was that I didn't want to mix new feature introduction (i.e. MBF21) with the bug fixes that I wanted to get into the 5.1.0 release.

Reply:
Ok, we continue as usual, will I do about 10 PRs related to mbf21?

Reply:
As you like, the first step is done.